package.json

```diff
@@ -295,7 +296,7 @@
     "tslint-config-prettier": "^1.15.0",
     "tslint-config-standard": "^8.0.1",
     "tslint-react": "^3.6.0",
-    "uglifyjs-webpack-plugin": "^2.1.1",
+    "uglifyjs-webpack-plugin": "1.3.0",
```

Comment:
This was acting up due to react-relay-network-modern's `.mjs` files. Left it at the version that Webpack was using before it replaced it [in the core](https://github.com/webpack/webpack/releases/tag/v4.26.0)